pkgs/applications/networking/newsreaders/pan/default.nix

```diff
@@ -1,24 +1,40 @@
 { spellChecking ? true
-, lib, stdenv, fetchurl, pkg-config, gtk3, gtkspell3 ? null
-, gmime2, gettext, intltool, itstool, libxml2, libnotify, gnutls
-, makeWrapper, gnupg
-, gnomeSupport ? true, libsecret, gcr
+, lib
+, stdenv
+, fetchurl
+, pkg-config
+, gtk3
+, gtkspell3
+, gmime2
+, gettext
+, intltool
+, itstool
+, libxml2
+, libnotify
+, gnutls
+, makeWrapper
+, gnupg
+, gnomeSupport ? true
+, libsecret
+, gcr
 }:
 
-assert spellChecking -> gtkspell3 != null;
-
-let version = "0.146"; in
-
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "pan";
-  inherit version;
+  version = "0.146";
 
   src = fetchurl {
     url = "http://pan.rebelbase.com/download/releases/${version}/source/pan-${version}.tar.bz2";
     sha256 = "17agd27sn4a7nahvkpg0w39kv74njgdrrygs74bbvpaj8rk2hb55";
   };
 
+  patches = [
+    # Take <glib.h>, <gmime.h>, "gtk-compat.h" out of extern "C"
+    ./move-out-of-extern-c.diff
```

Comment:
Can this be upstreamed?

Reply:
Work was in progress in [MR #14](https://gitlab.gnome.org/GNOME/pan/-/merge_requests/14), the diff was taken from them, with additional removals of `extern "C"` since it still wouldn't compile.